solver/src/liquidity/baseline_liquidity.rs

```diff
@@ -0,0 +1,10 @@
+use crate::liquidity::{AmmOrder, LimitOrder};
+use anyhow::Result;
+
+#[async_trait::async_trait]
+pub trait BaselineLiquidity {
+    async fn get_liquidity(
+        &self,
+        offchain_orders: &mut (dyn Iterator<Item = &LimitOrder> + Send + Sync),
+    ) -> Result<Vec<AmmOrder>>;
```

Comment:
Similar question to above (with regards to `Pool`). Will BalancerLiquidity be modelled as an AmmOrder or as some more specific type of liquidity?

Reply:
```suggestion
    ) -> Result<Vec<Liquidity>>;
```